### PR TITLE
Homework #2

### PR DESCRIPTION
**Pull request by weagleweagle**

### CODE REVIEW

File: lesson_02_homework.rb [hunk 1]
```diff
@@ -19,17 +29,31 @@ def fizz_buzz_to(limit)
...
+  def initialize(list)
+    @list = list
+  end
+
+  def add_party(party_name)
+    @list.push(party_name)
+    party_name
+  end
+
+  def list
```
Comment (kareemgrant) [inline lesson_02_homework.rb]: @efabery-handy  while this works `attr_accessor` has already done this for you, remember that `attr_accessor` is a shortcut that creates both a reader and a writer method

the reader method that `attr_accessor` creates looks like this:

```
def list
  @list
end
```
so in this case you are essentially defining the same method twice.

File: lesson_02_homework.rb [hunk 2]
```diff
@@ -19,17 +29,31 @@ def fizz_buzz_to(limit)
...
+    @list
+  end
+
+  def seat
+    return @list, @list.delete_at(0)
```
Comment (kareemgrant) [inline lesson_02_homework.rb]: :hammer: @weagleweagle this method returns both the list and the deleted element, the directions call for this method to only return the update list after a party has been seated.

**Comment by weagleweagle** [comment]
@kareemgrant

**Comment by kareemgrant** [comment]
:+1: @weagleweagle Approved

**Pull request closed without merging**